akka-actor-typed/src/main/scala/akka/actor/typed/internal/ExtensionsImpl.scala

```diff
@@ -49,7 +49,9 @@ trait ExtensionsImpl extends Extensions { self: ActorSystem[_] =>
             else throw new RuntimeException(s"[$extensionIdFQCN] is not an 'ExtensionId'")
           case Failure(problem) =>
             if (!throwOnLoadFail)
-              log.error(problem, "While trying to load extension [{}], skipping...", extensionIdFQCN)
+              log.error(
+                "Has been problem with {}, While trying to load extension [{}], skipping...",
+                Array(problem, extensionIdFQCN))
```

Comment:
If we should include `problem` it should be as the log.error exception rather than in the log string so that a stacktrace gets logged etc.

Reply:
you're right